operator/internal/manifests/internal/config/loki-runtime-config.yaml

```
@@ -73,6 +73,52 @@ overrides:
     {{- end }}
     {{- end}}
   {{- end -}}
+  {{- if $l := $spec.OTLP }}
+    otlp_config:
+      {{- with $l.ResourceAttributes }}
+      resource_attributes:
+        ignore_defaults: {{ .IgnoreDefaults }}
+        {{- with .Attributes}}
+        attributes_config:
+        {{- range . }}
+        - action: {{ .Action }}
+          {{- with .Regex }}
+          regex: {{ . }}
+          {{- end }}
+          attributes:
+          {{- range .Attributes }}
+          - {{ . }}
+          {{- end }}
+        {{- end }}
+        {{- end }}
+      {{- end}}
+      {{- with $l.ScopeAttributes }}
+      scope_attributes:
+      {{- range . }}
+      - action: {{ .Action }}
+        {{- with .Regex }}
+        regex: {{ . }}
+        {{- end }}
+        attributes:
+        {{- range .Attributes }}
+        - {{ . }}
+        {{- end }}
```

`with` for attributes:

```suggestion
        {{- with .Attributes }}
        attributes:
        {{- range . }}
        - {{ . }}
        {{- end }}
        {{- end }}
```